### PR TITLE
[DOC] Create documentation of open topics

### PR DESCRIPTION
**Pull request by m273d15**
This documentation references interesting PRs
which are closed, because they are not complete
or the approach was not usable.

### CODE REVIEW

File: docs/contribute/open-topics.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+---
+title: Open Topics
+---
+
+Here you can find:
+* information about topics which are started, but not completed
+* ideas and implementations which are never integrated into the main branch, but might be interesting.
```
Comment (tobous) [inline docs/contribute/open-topics.md]:
```suggestion
* ideas and implementations which were never integrated into the main branch, but might still be of interest.
```

File: docs/contribute/open-topics.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+Therefore this page can be used for finding open and interesting topics which can be continued.
+Please, let us know (e.g. via Gitter) before you continue a topic so we can share our thoughts and experiences.
```
Comment (tobous) [inline docs/contribute/open-topics.md]:
```suggestion
Please, let us know (e.g. via our [Gitter](https://gitter.im/saros-project/saros) or [mailing list](https://groups.google.com/forum/#!forum/saros-devel)) before you continue a topic so we can share our thoughts and experiences.
```

**Comment by m273d15** [comment]
I would like to merge this PR and just open an issue (and assign @stefaus).

File: docs/contribute/open-topics.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
+---
+title: Open Topics
+---
+
+Here you can find:
+* information about topics which are started, but not completed
```
Comment (tobous) [inline docs/contribute/open-topics.md]:
```suggestion
* information about topics which were started, but not completed
```

**Comment by tobous** [comment]
@m273d15 I think this can also just be merged directly as it only deals with the website and provides information immediately useful to potential contributors.

**Pull request merged**